src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -5,9 +5,13 @@ import Tab from '@material-ui/core/Tab'
 import Tabs from '@material-ui/core/Tabs'
 import Typography from '@material-ui/core/Typography'
 import gql from 'graphql-tag'
```

Comment:
I think we should break the workspaces (index cards) routes and the workspace route into two different files. This one is too big now

Reply:
Agreed, though it might be best as a separate issue to manage the scope of these changes.

Reply:
That's fair. I think it would be a good idea to prioritize this before adding more functionality to workspaces going forward